<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>testmodule5</class>
 <widget class="QDialog" name="testmodule5">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>390</width>
    <height>482</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>20</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:18pt; font-weight:600;&quot;&gt;C&lt;/span&gt;&lt;span style=&quot; font-size:18pt; font-weight:600; color:#ff0000;&quot;&gt;++&lt;/span&gt;&lt;span style=&quot; font-size:18pt; font-weight:600;&quot;&gt; Test&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>331</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Wann wird eine Switch-Case einer If-Anweisung &lt;br&gt;vorgezogen?</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>361</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Wenn es nur eine zu überprüfende Variable gibt.</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>160</y>
     <width>371</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Wenn nur Zahlenwerte überprüft werden sollen.</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>190</y>
     <width>371</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Wenn Integer Variabken überprüft werden sollen.</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>-40</x>
     <y>210</y>
     <width>441</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>230</y>
     <width>371</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Zeige mithilfe der Switch-Case an, welchen Wert &lt;br/&gt;die Variable &amp;quot;b&amp;quot; hat. Das Label soll b=? anzeigen. &lt;br/&gt;(? = gesuchte Zahl)&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>290</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>int b = 3;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="tb1">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>310</y>
     <width>113</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>290</y>
     <width>67</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;{&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="tb2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>340</y>
     <width>113</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="tb3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>370</y>
     <width>113</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="tb4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>400</y>
     <width>113</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>340</y>
     <width>131</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>ui-&gt;label-&gt;setText</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>370</y>
     <width>131</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>ui-&gt;label-&gt;setText</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>400</y>
     <width>131</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>ui-&gt;label-&gt;setText</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="tb5">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>340</y>
     <width>71</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="tb6">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>370</y>
     <width>71</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="tb7">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>400</y>
     <width>71</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>340</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>(&quot;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>340</y>
     <width>31</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>&quot;)</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>370</y>
     <width>21</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>(&quot;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>370</y>
     <width>31</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>&quot;)</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_14">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>400</y>
     <width>21</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>(&quot;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_15">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>400</y>
     <width>31</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>&quot;)</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>450</y>
     <width>99</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Bestätigen</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
